DigitalLearningSolutions.Web/ViewModels/MyAccount/UpdateNotificationPreferencesViewModel.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.MyAccount
+{
+    using System.Collections.Generic;
+    using DigitalLearningSolutions.Data.Models;
+
+    public class UpdateNotificationPreferencesViewModel
+    {
+        public UpdateNotificationPreferencesViewModel(
+            IEnumerable<NotificationPreference> notifications, string userType)

```

Comment:
Super NIP, but given that the params are split on new lines, could we also put "userType" on a new line? 🙏